### PR TITLE
I guess... STUFFF

### PR DESCRIPTION
**Pull request by Fortifier42**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java [hunk 1]
```diff
@@ -114,6 +114,26 @@ public void serverTag(ReplaceableTagEvent event) {
         Attribute attribute = event.getAttributes().fulfill(1);
 
         // <--[tag]
+        // @attribute <server.ip>
+        // @returns Element
+        // @description
+        // returns the server IP Address.
+        // -->
+        if (attribute.startsWith("ip")) {
+            event.setReplaced(new Element(Bukkit.getServer().getIp()).getAttribute(attribute.fulfill(1)));
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: So young, so innocent...

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: So @BlackCoyote

File: src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java [hunk 2]
```diff
@@ -114,6 +115,31 @@ public void serverTag(ReplaceableTagEvent event) {
...
 
         // <--[tag]
+        // @attribute <server.ip>
+        // @returns Element
+        // @description
+        // returns the server IP Address.
+        // -->
+        if (attribute.startsWith("ip")) {
+            try {
+                event.setReplaced(new Element(InetAddress.getLocalHost().getHostAddress()).getAttribute(attribute.fulfill(1)));
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: could you not

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: this is v old D:

Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: it's a DAY OLD and part of your current pr

**Pull request merged**